lib/charms/mongodb/v0/config_server_interface.py

```diff
@@ -32,7 +35,7 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 2
+LIBPATCH = 5
```

Comment:
Are you 100% sure that this jump in library version will be ok with charmhub?

Reply:
please check the library file. This file is not owned by `mongos` and we do not publish it on this charm